libraries/ms-common/src/stm32f0xx/pwm.c

```diff
@@ -3,19 +3,69 @@
 #include <stdint.h>
 
 #include "gpio.h"
+#include "pwm_mcu.h"
 #include "stm32f0xx_rcc.h"
 #include "stm32f0xx_tim.h"
 
-static uint16_t s_period_ms = 0;
+static uint16_t s_period_ms[NUM_PWM_TIMERS] = {
```

Comment:
You could use `= { 0 }` if you wanted to. If you feel this is more explicit I'm fine with that.